bee-network/src/timing.rs

```diff
@@ -0,0 +1,30 @@
+pub(crate) const DEFAULT_RECONNECT_INTERVAL: Seconds = Seconds(60);
```

Comment:
This is a helper file, I don't think this default should be here.

Reply:
Also, I feel like this file coule be in bee-common.

Reply:
Removed it entirely.